test/local/account_routes.js

```diff
@@ -90,7 +90,14 @@ test(
   'account with verified invalid email does not get deleted on status poll',
   function (t) {
     var mockDB = {
-      deleteAccount: sinon.spy()
+      deleteAccount: sinon.spy(),
+      sessionTokenWithVerificationStatus: function () {
+        return P.resolve({
+          email: TEST_EMAIL_INVALID,
+          tokenVerified: true,
+          emailVerified: true
+        })
```

Comment:
Does this mock match the return signature of `sessionTokenWithVerificationStatus`?  I thought it returned the token with e.g. `tokenVerificationId` property and we had to check that to determine whether the token was verified.